src/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface genesisStatesType {
+  names: {
+    [key: string]: string
```

Comment:
It probably doesn't matter, but is there a particular reason why the key here and in `chainsType` is `string` and not `number`? `Common.setChain` seems to be using it with `number`.

Reply:
JavaScript seems to implicitly convert such `number` assignments to `string` - also figured out during experimentation while working on this 😄 - e.g.:

```shell
> var t = {}
undefined
> t[4] = 'test'
'test'
> t
{ '4': 'test' }
```